app/Events/RestoreJobFinished.php

```
@@ -17,17 +17,23 @@ public function __construct($data)
+            $commands = [];
+
+            if (isSafeTmpPath($scriptPath)) {
+                $commands[] = "docker exec {$container} sh -c 'rm ".escapeshellarg($scriptPath)." 2>/dev/null || true'";
+            }
+
+            if (isSafeTmpPath($tmpPath)) {
+                $commands[] = "docker exec {$container} sh -c 'rm ".escapeshellarg($tmpPath)." 2>/dev/null || true'";
+            }
+
+            if (! empty($commands)) {
+                $server = Server::find($serverId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Server' in method '__construct'. (undefined)

(StaticAccess)